razzle.config.js

```diff
@@ -54,6 +57,23 @@ module.exports = {
       };
     }
 
+    if (target === 'web') {
+      appConfig.plugins.push(
+        new OfflinePlugin({
+          appShell: '/article/',
```

Comment:
Have you seen this? https://github.com/NekR/offline-plugin/blob/abdae89fd777ed7e6ea05ad31c993ffaf3593b5b/docs/app-shell.md#ssr. I would potentially recommend removing this config value for now.